src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -755,10 +757,15 @@ def call(
             query_vectors, key_vectors, self.one_sided_attn_window_size
         )
 
+        # values to pad for attention probs
+        remove_from_windowed_attention_mask = attention_mask != 0
+        # cast to fp32/fp16 then replace 1's with -inf
+        float_mask = tf.cast(remove_from_windowed_attention_mask, dtype=query_vectors.dtype) * LARGE_NEGATIVE
+
```

Comment:
This is not logically necessary, since `TFLongformerMainLayer` prepares `attention_mask` slightly different from `LongformerModel`

PT
https://github.com/huggingface/transformers/blob/d7136abe59f2183ab4efb343ae249b089be83681/src/transformers/models/longformer/modeling_longformer.py#L1688

TF (note the `abs`)
https://github.com/huggingface/transformers/blob/d7136abe59f2183ab4efb343ae249b089be83681/src/transformers/models/longformer/modeling_tf_longformer.py#L1745

Need to keep this for `# copied from` in `TFLEDEncoderSelfAttention`, because for that model, this block is necessary - the reason is: `tf.abs` is not used when preparing the attention mask there

https://github.com/huggingface/transformers/blob/d7136abe59f2183ab4efb343ae249b089be83681/src/transformers/models/led/modeling_tf_led.py#L103